systemtest/src/main/java/io/strimzi/systemtest/annotations/NetworkPolicyCondition.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.annotations;
+
+import io.strimzi.systemtest.Environment;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.extension.ConditionEvaluationResult;
+import org.junit.jupiter.api.extension.ExecutionCondition;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+public class NetworkPolicyCondition implements ExecutionCondition {
+    private static final Logger LOGGER = LogManager.getLogger(NetworkPolicyCondition.class);
+
+    @Override
+    public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext extensionContext) {
+        if (Environment.NETWORK_POLICY.equals("true")) {
+            return ConditionEvaluationResult.enabled("Test is enabled");
+        } else {
+            LOGGER.info("{} is for NetworkPolicy, but the running cluster has not enabled NetworkPolicy {}",
+                    extensionContext.getDisplayName(),
+                    extensionContext.getDisplayName()
```

Comment:
Are you displaying the same information twice?

Reply:
Thanks for catch. Message could a little bit confusing i added description

Reply:
@Seequick1 can you just use "Ignoring it" instead of repeating the name two times? Wdyt?